extensions/vale-styleguide/README.md

```diff
@@ -0,0 +1,33 @@
+# suse-vale-styleguide README
+
+## Features
+
+The SUSE Vale Styleguide extension includes Vale rules that the styleguide of
+the SUSE documentation team uses.
+
+## Installation
+
+For this extension to work, meet the following requirements:
+
+1. Download the `vale` binary from
+   [github.com/errata-ai/vale/releases](https://github.com/errata-ai/vale/releases),
+   unpack it and copy to a directory in your `$PATH`.
+2. Install the `vale-vscode` VSCode extension from the VSCode market place
+    ```
+    ext install ChrisChinchilla.vale-vscode
```

Comment:
I don't have the `ext` executable nor do I know where to get it. Perhaps for others is also a big problem. However, most have installed `code`.

Would it be an option to install the extension like this:

```
code --install-extension ChrisChinchilla.vale-vscode
```

I've tried it and it gives me this output:

```
$ code --install-extension ChrisChinchilla.vale-vscode
Installing extensions...
Installing extension 'chrischinchilla.vale-vscode'...
(node:31365) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `code --trace-deprecation ...` to show where the warning was created)
Extension 'chrischinchilla.vale-vscode' v0.17.3 was successfully installed.
```

```suggestion
    code --install-extension ChrisChinchilla.vale-vscode
```